app/src/main/java/app/revanced/manager/ui/component/bundle/BundlePatchesDialog.kt

```diff
@@ -36,13 +35,13 @@ fun BundlePatchesDialog(
     bundle: PatchBundleSource,
 ) {
     var informationCardVisible by remember { mutableStateOf(true) }
+    var showAllVersions by remember { mutableStateOf(false) }
+    var showOptions by remember { mutableStateOf(false) }
```

Comment:
This state will be lost on configuration changes such as screen rotation. You probably need to use `rememberSaveable` here.

`informationCardVisible` should probably also be `rememberSaveable`.